### DISCUSSION TITLE
[Feature]: prerender plugin

### DISCUSSION THREAD

**Discussion by reaink** [web-infra-dev/rsbuild#3474] [General]
### What problem does this feature solve?

static generator uplift seo, first load time

### What does the proposed API look like?

a new plugin

**Comment by chenjiahan** [comment]
Hi, you can refer to: https://github.com/web-infra-dev/rsbuild/discussions/2917

**Comment by reaink** [comment]
> Hi, you can refer to: https://github.com/web-infra-dev/rsbuild/discussions/2917

Waiting for the official Prerender plugin
